src/Interfaces/SharedInterfaces.tsx

```diff
@@ -0,0 +1,3 @@
+export interface AssignmentProps {
```

Comment:
Please move it to screens or component level as you have done it for props and styles.